server/application/application.go

```diff
@@ -745,7 +745,7 @@ func (s *Server) Get(ctx context.Context, q *application.ApplicationQuery) (*v1a
 	})
 	defer unsubscribe()
 
-	app, err := argo.RefreshApp(appIf, appName, refreshType)
+	app, err := argo.RefreshApp(appIf, appName, refreshType, true)
```

Comment:
Any impact to set the hydrate annotation on applications that are not configured with the hydrator?

Will the controller detect that this app is not configured for hydration and remove the annotation?